Source/Core/Core/HW/EXI/EXI_DeviceAD16.h

```diff
@@ -28,7 +28,7 @@ class CEXIAD16 : public IEXIDevice
   union AD16Reg
   {
     u32 U32 = 0;
-    u32 U8[4];
+    u8 U8[4];
```

Comment:
Doesn't this change require you to bump the savestate version?

Reply:
Thanks, fixed